litestar/_openapi/schema_generation/utils.py

```diff
@@ -107,3 +107,14 @@ def get_formatted_examples(field_definition: FieldDefinition, examples: Sequence
     name = name.lower()
 
     return {f"{name}-example-{i}": example for i, example in enumerate(examples, 1)}
+
+
+def get_json_schema_examples(field_definition: FieldDefinition, examples: Sequence[Example | str]) -> list[Any]:
```

Comment:
```suggestion
def get_json_schema_examples(examples: Sequence[Example | str]) -> list[Any]:
```
Wonder why linters didn't get this.

Reply:
Ha, indeed...